metricbeat/module/beat/stats/data.go

```diff
@@ -76,8 +76,29 @@ var (
 			"pipeline": c.Dict("pipeline", s.Schema{
 				"clients": c.Int("clients"),
 				"queue": c.Dict("queue", s.Schema{
-					"acked":      c.Int("acked"),
 					"max_events": c.Int("max_events"),
```

Comment:
`queue.max_events` appears to also have been dropped, but not added to the backwards compatibility list, like `acked`?